grails-app/controllers/org/pih/warehouse/api/ProductApiController.groovy

```diff
@@ -274,4 +275,13 @@ class ProductApiController extends BaseDomainApiController {
         render([monthlyDemand: demand.monthlyDemand, quantityOnHand: quantityOnHand] as JSON)
     }
 
+    def getInventoryItem = {
+        InventoryItem inventoryItem = InventoryItem.findByLotNumberAndProduct(params.lotNumber, Product.findByProductCode(params.productCode))
+        if (inventoryItem) {
+            Integer quantityOnHand = productAvailabilityService.getQuantityOnHand(inventoryItem)
+            render([inventoryItem: inventoryItem, quantityOnHand: quantityOnHand] as JSON)
```

Comment:
This is fine for now, but ...

I'd like to return a single inventory item JSON object.
```
inventoryItem.quantityOnHand = productAvailabilityService.getQuantityOnHand(inventoryItem)
render([inventoryItem:inventoryItem])
```
This would require a small refactoring that I don't think I want to do right now. 
InventoryItem.toJson() already includes the quantities, but Bootstrap.groovy defines its own marshaller for InventoryItem, so we'd need to fix the quantities returned by InventoryItem.toJson() method and also register the improved method as the default object marshaller. 

```
JSON.registerObjectMarshaller(InventoryItem) { InventoryItem inventoryItem ->
    return inventoryItem.toJson()
}
```